component/Badges/styles.css

```diff
@@ -0,0 +1,175 @@
+
+@import url('/component/NavBar-Vertical/styles.css');
+@import url('/styles/styles-global/root.css');
+@import url('/styles.css');
+@import url('/component/Avatars/styles.css');
+
+
+.avatar-rd-img-container,
+.avatar-squared-img-container,
+.avatar-rd-txt-container,
+.avatar-sq-txt-container
+{
+    width: 90%;
+    height: 8rem;
+}
+
+
+
+
+
+/* ROUND BADGE */
+
+.badge-container{
+    overflow: initial;
+    position: relative;
+}
+
+.bg-online{
+    background-color: red;
+}
+
+.bg-available{
+    background-color: green;
+}
+
+.bg-away{
+    background-color: yellow;
+}
+
+.bg-left{
+    left: 0;
+    
+}
+
+.bg-right{
+    right: 0;
+}
+
+.badge-round{
+    border-radius: 50%;
+}
+
+.badge-square{
+    border-radius: 30%;
+}
+
+
+.badge-xl,
+.badge-l,
+.badge-m,
+.badge-s,
+.badge-xs
+{
+    position: absolute;
+    bottom: 0;
+}
+
+.badge-xl{
+    width: 2rem;
+    height: 2rem;
+    border: 8px solid var(--white-color);
+}
+
+.badge-l{
+    width: 1.25rem;
+    height: 1.25rem;
+    border: 5px solid var(--white-color);
+}
+
+.badge-m{
+    width: 1rem;
+    height: 1rem;
+    border: 4px solid var(--white-color);
+}
```

Comment:
since the border color is same it can be added on the above line, rest of the code looks good can be merged

Reply:
Done. Thank you